packages/cashier/src/Containers/__tests__/deposit.spec.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Deposit from '../deposit';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+jest.mock('Components/Error/virtual', () => () => <div>Virtual</div>);
+jest.mock('Components/Error/cashier-locked', () => () => <div>CashierLocked</div>);
+jest.mock('Components/Error/funds-protection', () => () => <div>FundsProtection</div>);
+jest.mock('Components/Error/deposit-locked', () => () => <div>DepositsLocked</div>);
+jest.mock('Components/Form/crypto-transactions-history', () => () => <div>CryptoTransactionsHistory</div>);
+jest.mock('Components/Error/error', () => () => <div>Error</div>);
+jest.mock('../crypto-deposit', () => () => <div>CryptoDeposit</div>);
+jest.mock('Components/cashier-container', () => () => <div>CashierContainer</div>);
+jest.mock('Components/CashierDefault/cashier-default', () => () => <div>CashierDefault</div>);
+
+describe('<Deposit />', () => {
+    const mockDefaultProps = () => ({
```

Comment:
can we assign this directly to an object so we don't need to re-assign it to another variable like on line 47?

Reply:
sure, thx for noticed